cluster/images/provider-template-controller/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM BASEIMAGE
```

Comment:
Should we make the same changes from https://github.com/crossplane/crossplane/pull/2718 ?

Reply:
I have a PR https://github.com/crossplane-contrib/provider-kafka/pull/20 that migrates another provider-template based project onto the new makefile. Perhaps we can use it as a template?

Reply:
I'd prefer to mostly propagate what we do today in this PR and make changes in future PRs. So let's keep this for now.